<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreCargarNomina</class>
 <widget class="QDialog" name="PreCargarNomina">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/atras.png</normaloff>:/atras.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCargar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/abrir.jpeg</normaloff>:/abrir.jpeg</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>10</y>
     <width>251</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>AÃ±o...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbAno"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listNominas"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
